src/test/kotlin/org/malachite/estella/queue/QueueExampleIntegration.kt

```diff
@@ -0,0 +1,26 @@
+package org.malachite.estella.queue
+
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.malachite.estella.commons.models.offers.StageType
+import org.malachite.estella.queues.ExampleConsumer
+import org.malachite.estella.queues.ExamplePublisher
+import org.springframework.http.HttpStatus
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+import strikt.assertions.isNotNull
+import java.lang.Thread.sleep
+
+class QueueExampleIntegration {
+
+    @Test
+    @Order(1)
+    fun `Example consumer should receive 1 message`() {
```

Comment:
Ten test Ci zadziałał? W sensie z tego, co ja wiem, to musisz mieć odpalonego tego rabbita. Miałeś? Bo na przykład my możemy nie mieć i nam wtedy ten test raczej nie przejdzie i będziemy się zastanawiać dlaczego. Ogólnie do czegoś takiego służy test containers. Jesli z nich skorzystamy, że łatwiej będzie pobierać amqp URL z @Value co Ci wcześniej pisałem w komentarzach, więc tam lepiej też zmień.

Reply:
Test przeszedł bo rabbit nie jest odpalony u nas tylko na cloud amqp XDDD

Reply:
Dobra ogólnie zapomniałem, że się tam połączyłeś do cloud AMQP. Ale still nie wiem,  czy to dobry pomysł. Miliona wiadomości pewnie nie wykorzystamy, ale chyba lepiej dmuchać na zimne. Plus o test containers możemy napisać w domu, bo jest to coś niestandardowego XDDD